LibVLCSharp.Uno/LibVLCSharp.Uno.csproj

```diff
@@ -46,19 +49,32 @@ It also contains a VLC MediaPlayerElement for the Uno Platform (UWP, Android, iO
     <None Include="**\*.UWP.cs" />
     <None Include="**\*.Android.cs" />
     <None Include="**\*.iOS.cs" />
+    <Page Remove="**\*.UWP.xaml" />
+    <Page Remove="**\*.Android.xaml" />
+    <Page Remove="**\*.iOS.xaml" />
+    <None Include="**\*.UWP.xaml" />
+    <None Include="**\*.Android.xaml" />
+    <None Include="**\*.iOS.xaml" />
   </ItemGroup>
   <ItemGroup Condition="$(TargetFramework.StartsWith('xamarinios')) or $(TargetFramework.StartsWith('monoandroid'))">
-    <PackageReference Include="Uno.UI" Version="2.0.512-dev.3599" />
+    <PackageReference Include="Uno.UI" Version="2.0.512-dev.3644" />
   </ItemGroup>
   <ItemGroup Condition="$(TargetFramework.StartsWith('uap'))">
-    <Compile Remove="Themes\VideoView.xaml" />
     <Compile Include="**\*.UWP.*cs" />
+    <Page Include="**\*.UWP.*xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
   </ItemGroup>
   <ItemGroup Condition="$(TargetFramework.StartsWith('monoandroid'))">
     <ProjectReference Include="..\LibVLCSharp.Android.AWindow\LibVLCSharp.Android.AWindow.csproj" />
     <Compile Include="**\*.Android.*cs" />
+    <Page Include="**\*.Android.*xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
   </ItemGroup>
   <ItemGroup Condition="$(TargetFramework.StartsWith('xamarinios'))">
     <Compile Include="**\*.iOS.*cs" />
+    <Page Include="**\*.iOS.*xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.ComponentModel.DataAnnotations">
+      <HintPath>C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\Xamarin.iOS\v1.0\System.ComponentModel.DataAnnotations.dll</HintPath>
```

Comment:
This will be a problem for building on macOS I'm afraid.

Reply:
Visual Studio does some weird stuff sometimes. Thank you, I delete it.